shinhye/db/migrations/20230203013024_create_like_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE likes (
+  id int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  user int NOT NULL,
+  post int NOT NULL,
```

Comment:
각각의 컬럼은 외래키로 user와 post의 정보를 받기 때문에 user_id와 post_id로 수정해주세요

```suggestion
  user_id int NOT NULL,
  post_id int NOT NULL,
```